motion:

  start_time: 0
  soft_off_time: 30
  die_time: 30
  anim_server_wait_time: 1

  animations:
    walkready : "walkready" # Gibt eine Dynamische Walkready #todo change to dynamic pose from walking, like it was before
    goalie-walkready: "goalie-walkready" # Gibt eine Dynamische Walkready #todo change to dynamic pose from walking, like it was before
    front-up : "front-up"
    bottom-up : "bottom-up"
    toSquat: "aufrichten-hocke-tamara"
    stand-up: "stand-up"
    penalized : "sit-down"
    sit-down : "sit-down"

  falling:

    #falling animations
    "falling_front": "falling-front"
    "falling_back": "falling_back"
    "falling_left": "falling_left"
    "falling_right": "falling_right"

    #Werte erstellt durch 3doerfle, 3bimberg, 3windsor (Praktikum sose 2015)
    "Darwin":

      #Grenzwerte müssen auf härterem boden auch härter eingestellt werden,
      #dies erfolgt mit dem boden_koeffizient
      "ground_coefficient": 1.25   #WEICH(kunstrasen Spielfeld)= 1.0 , HART(teppich Spielfeld)= 1.25

      #Grenzwerte schärfer --> fruehere Reaktion aber haeufigere Fehlinterpretation, zb. beim Laufen
      #Grenzwerte sanfter --> zu spaete Raktion
      "threshold_gyro_y_front":  -50.0 # > gyroY
      "threshold_gyro_y_back":  50.0 # < gyroY
      "threshold_gyro_x_right": -30.0 # > gyroX
      "threshold_gyro_x_left":   30.0 # < gyroX

      #false = motoren werden bei fallerkennung abgeschaltet
      "dyn_falling_active": true


    "Hambot": #Achtung werte nur kopiert und nicht getestet!!!

      #Koeffizient wird auch benutzt wenn fallen deaktiviert --> nur dass er die motoren ausmacht
      "ground_coefficient": 1.25 #--> wert ist gut für HAMBOT, da dieser Wert das laufen nicht beeinträchtigt

      "threshold_gyro_y_front":  -22 # > gyroY
      "threshold_gyro_y_back":  22 # < gyroY
      "threshold_gyro_x_right": -30 # > gyroX
      "threshold_gyro_x_left":   30 # < gyroX

      #false = motoren werden bei fallerkennung abgeschaltet
      "dyn_falling_active": false

    "Minibot":

      #Grenzwerte müssen auf härterem boden auch härter eingestellt werden,
      #dies erfolgt mit dem boden_koeffizient
      "ground_coefficient": 1.5  #WEICH(kunstrasen Spielfeld)= 1.0 , HART(teppich Spielfeld)= 1.25

      #Grenzwerte schärfer --> fruehere Reaktion aber haeufigere Fehlinterpretation, zb. beim Laufen
      #Grenzwerte sanfter --> zu spaete Raktion
      "threshold_gyro_y_front":  -50.0 # > gyroY
      "threshold_gyro_y_back":  50.0 # < gyroY
      "threshold_gyro_x_right": -30.0 # > gyroX
      "threshold_gyro_x_left":   30.0 # < gyroX

      #false = motoren werden bei fallerkennung abgeschaltet
      "dyn_falling_active": true



